frontend/src/metabase/visualizations/visualizations/PivotTable.css

```diff
@@ -0,0 +1,6 @@
+:local .PivotBGLight {
```

Comment:
If possible I'd like to not keep creating new css modules code so we can eventually rip it out and get rid of yet another CSS solution. I'd just define two new variables in the Pivot Table file and use our `metabase/lib/color` library there to get this effect.